apps/paf/translations/src/en/fields.json

```diff
@@ -587,5 +587,95 @@
   },
   "personAddNi": {
     "label": "National insurance number or equivalent (if known)"
+  },
+  "about-you": {
+     "label": "How did you find out about the crime?"
```

Comment:
would it make sense to change the field name to match more the field question? e.g. `how-did-you-find-out-about-the-crime`